gradle/root/test.gradle

```diff
@@ -327,10 +327,23 @@ task parallelCombinedTestReport(type: TestReport) { t ->
  *********************************************************************************/
 task unitTestReport(type: TestReport) { t ->
 	group "test"
-	destinationDir = file("$reportDir/unitTests")
+    description "Run unit tests and save HTML report."
+    destinationDir = file("$reportDir/unitTests")
 	outputs.upToDateWhen {false}
 }
 
+/*********************************************************************************
+ * INTEGRATION TEST REPORT
+ *
+ * Summary:	Runs all integration tests and generates a single report.
+ *
+ *********************************************************************************/
+task integrationTestReport(type: TestReport) { t ->
+	group "test"
+    description "Run unit integrtion tests and save HTML report."
```

Comment:
Small misspelling fix, and remove "unit":

"Run integration tests and save...."